netutils/ping/icmpv6_ping.c

```diff
@@ -57,12 +59,22 @@
  * separate instance of g_ping6_id in every process space.
  */
 
-static uint16_t g_ping6_id = 0;
+static uint16_t g_ping6_id;
+static volatile bool exiting;
```

Comment:
```suggestion
static volatile bool g_exiting;
```

Reply:
Static variables cannot be repeated with those in ping.c, so they are modified to g_exiting6